debian/patches/drop-azure-tests.patch

```diff
@@ -0,0 +1,308 @@
+Description: Remove Azure tests.
```

Comment:
Realized after the fact that you might be able to just skip these tests in the `rules` file

```
override_dh_auto_test:
	PYTHONPATH=. py.test --ignore=tests/test_wabs_deleter.py --ignore=tests/test_wabs...
```

https://pytest.org/latest/example/pythoncollection.html